.github/workflows/windows.yml

```diff
@@ -0,0 +1,44 @@
+name: windows
+
+on:
+  pull_request:
+  push:
+  release:
+    types: published
+
+jobs:
+  build:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        buildtype: [Debug, Release]
+        standalone: [ON, OFF]
+        platform: [Win32, x64]
+        exclude:
+          - {buildtype: "Debug", standalone: "OFF", platform: x64}
+          - {buildtype: "Release", standalone: "OFF", platform: x64}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+        
+    - name: Build Thyme
+      run: |
+        cmake -A ${{ matrix.platform }} -DSTANDALONE=${{ matrix.standalone }} -DUSE_CRASHPAD=${{ matrix.standalone }} -B build
```

Comment:
You can use ninja here as well, but that requires changing how the arch is passed. You get sane error messages and better build speed in turn. I will try providing a sample.

Reply:
Here's one for appveyor, I know it's not quite compatible, but it's good for starters: https://github.com/openblack/openblack/pull/68/files